### PR TITLE
Tls restore compat

### PR DESCRIPTION
**Pull request by huitema**
Removing the gratuitous change of QHKDF-Expand in draft 10, resolving issue #1255

### CODE REVIEW

**Comment by marten-seemann** [comment]
Would this be in draft-11, and would we use that for the remote interop?

File: draft-ietf-quic-tls.md [hunk 1]
```diff
@@ -703,12 +703,13 @@ structure:
 struct {
    uint16 length = Length;
    opaque label<6..255> = "QUIC " + Label;
+   uint8 hashLength = 0;
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md]: This needs to be explained, or people will ask why the hash is zero length.

**Comment by kazuho** [comment]
@martinthomson 
> For mine, a more principled AEAD API would take two labels: one for the key and one for the label, and avoid baking in the specific details of the TLS HKDF info structure. A hacky API change could distinguish between a NULL hash and an empty hash.
> 
> In summation, I'd prefer not to have to explain to future users of QUIC that this isn't a null terminator on the string, but a product of a design decision made before QUIC even really existed.

Thank you for your comments. While I understand and have sympathy to your arguments, I wonder if there's a strong reason to divert from the structure defined in TLS 1.3 considering the fact that the quic-tls draft depends on TLS 1.3.

**Comment by ekr** [comment]
It's not just PicoTLS. This is an issue for Minq as well.

**Comment by martinthomson** [comment]
picotls users, does h2o/picotls#138 mean that we're ok with closing this?

**Comment by huitema** [comment]
I am fine with closing this as "won't fix".

**Pull request closed without merging**